---
# 1 iff person is on path
byte value